config.js

```diff
@@ -26,13 +27,19 @@ module.exports = {
     password: process.env.REDIS_PASSWORD
   },
   email: {
-    caseworker: process.env.CASEWORKER_EMAIL || '',
     from: process.env.FROM_ADDRESS || '',
     replyTo: process.env.REPLY_TO || '',
-    accessKeyId: process.env.AWS_USER || '',
-    secretAccessKey: process.env.AWS_PASSWORD || '',
-    transportType: 'ses',
-    region: process.env.EMAIL_REGION || ''
+    region: process.env.EMAIL_REGION || '',
+    transport: process.env.EMAIL_TRANSPORT || 'ses',
+    caseworker: process.env.CASEWORKER_EMAIL || '',
+    transportOptions: {
+      accessKeyId: process.env.HOF_SES_USER || process.env.AWS_USER || '',
+      secretAccessKey: process.env.HOF_SES_PASSWORD || process.env.AWS_PASSWORD || '',
+      port: process.env.TRANSPORT_PORT || '',
+      host: process.env.TRANSPORT_HOST || '',
+      ignoreTLS: process.env.TRANSPORT_IGNORE_TLS || '',
+      secure: process.env.TRANSPORT_SECURE || ''
```

Comment:
We're using SES, so we don't need most of these options because they only apply for other transports.

Reply:
@sulthan-ahmed the emails TO/FROM will can in most instances be set in the server config (kube-rotm repo). Is you cannot do that but the ticket you are working form request it, could you create a new ticket with those details please?

Reply:
I guess I'm being selfish here.  I'm using smtp and so this would make my life a little easier.  I'm still locked out of Amazon AWS account.  I'll remove and just use locally

Reply:
I'd argue that you're possibly making your life more difficult rather than being selfish.

If you want smtp for local dev then that's A-OK, but personally I find it _way_ easier when changing config for local dev just to actually change the config file rather than adding an extra layer of abstraction by defining environment variables.

Also, if you are developing emails locally then you'll likely find that actually the best transport is the recently added `debug` which writes the html to a file and (optionally) opens it in a browser tab. See https://github.com/UKHomeOfficeForms/hof-emailer#debug

Reply:
what I was planning to do was set up a .r2r-env-variables file and just load it when I run r2r so I thought that would be quicker.

oooo `debug` sounds like the way to go.  I'll give that a try actually

Reply:
@sulthan-ahmed I have a plan for env vars in development. Watch this space...